scripts/linters/pylint_extensions.py

```diff
@@ -1804,3 +1845,4 @@ def register(linter):
     linter.register_checker(DocstringChecker(linter))
     linter.register_checker(BlankLineBelowFileOverviewChecker(linter))
     linter.register_checker(NewlineBelowClassDocstring(linter))
+    linter.register_checker(SingleLinePragmaChecker(linter))
```

Comment:
I think we should move this check to `SingleLineCommentChecker`? 

This way we would all the checks related to single line comments under one class.

```
def _check_inline_pragma (self, tokens, ..)
// Single line pragma code

def process_tokens()
  for (token_type, _, (line_num, _), _, line) in tokens:
     if token_type == tokenize.COMMENT:
       call _check_inline_praga()
       call _check_other checks


```

Reply:
Just as an FYI, you can have a tuple for implementation i.e, it can be (IRawChecker, ITokenChecker,)

Reply:
If there is a future TODO here, please file an issue and add a TODO comment pointing to that issue, otherwise this thread cannot be considered resolved.

Reply:
Yes.

Follow this procedure, **always**: for any deferred work, (a) file an issue, (b) add a link to that issue in this comment thread, and, (c) if the issue relates to a specific part of the code, add a TODO comment pointing to that issue (if applicable). Make sure the comment is of the form `TODO(#...):` where the part in the parentheses is the issue number.

Reply:
Created the issue (https://github.com/oppia/oppia/issues/10042)